### ISSUE TITLE
Depth index should not be called DEPT

### ISSUE THREAD

**Issue by kwinkunks** [agile-geoscience/welly#205]
`DEPT` is a LAS thing, and could sometimes be `TIME` or `DEPTH` (as well as DEPT... yes, it's gross). At least, those are the 3 options in the standard. I suppose a person could come up with other things (MD, TVDSS, etc).

We should probably regularize it to DEPTH or TIME, so at least it's somewhat predictable. `depth` or `time` might be even more convenient, but a person might possibly suppose that it's uppercase, based on the standard.

Calling it a bug, because it's a break from the `welly` philosophy of being natural and 'geological', not just mirroring the LAS standard (that's what `lasio` is for).

**Comment by ThomasMGeo** [comment]
Does it make sense to make it Depth_ft or Depth_m? Might be too weird.

**Comment by ThomasMGeo** [comment]
💯 ! Sounds good, that makes sense.

**Comment by patrick-reinhard** [comment]
If we go ahead with this, would we like to support a way that you can still export with the original index mnemonic (e.g. `DEPT`) ?

**Comment by kinverarity1** [comment]
Agreed - in lasio it's properly "index" i.e. first curve is `LASFile.index`, but I don't think that's a very natural term to a geologist.
